integration/setup/setup.go

```diff
@@ -334,7 +334,8 @@ func insertBenchmarkProvider(tb testtb.TB, ctx context.Context, collection *mong
 func setupUser(tb testtb.TB, ctx context.Context, client *mongo.Client) {
 	tb.Helper()
 
-	if IsMongoDB(tb) {
+	if IsPostgreSQL(tb) || IsMongoDB(tb) {
+		// the user is created in env tool for PostgreSQL
```

Comment:
For SQLite we need something similar to have a database with default user. Will be handle in a separate PR.

Reply:
What about for MySQL?

Reply:
Yes that's true. MySQL hasn't been handled. let me see the status of MySQL backend

Reply:
MySQL and HANA backends are not yet supported, no need to do anything for them